src/pybind/mgr/volumes/fs/async_cloner.py

```diff
@@ -217,13 +218,31 @@ def set_quota_on_clone(fs_handle, clone_volumes_pair):
         except cephfs.Error as e:
              raise VolumeException(-e.args[0], e.args[1])
 
+def update_clone_progress(fs_handle, clone_volumes, start_time):
+    st = fs_handle.statx(clone_volumes[0].path, cephfs.CEPH_STATX_BTIME,
+                         cephfs.AT_SYMLINK_NOFOLLOW)
+    clone_creation_time = st["btime"]
+    idle_time = start_time - clone_creation_time
+    total_idle_secs = str(idle_time.total_seconds()).split(".")[0]
+    clone_volumes[0].add_clone_progress(total_idle_secs,
+                                        start_time.isoformat(timespec='seconds'),
+                                        flush=True)
+
 def do_clone(fs_client, volspec, volname, groupname, subvolname, should_cancel):
     with open_volume_lockless(fs_client, volname) as fs_handle:
         with open_clone_subvolume_pair(fs_client, fs_handle, volspec, volname, groupname, subvolname) as clone_volumes:
             src_path = clone_volumes[1].snapshot_data_path(clone_volumes[2])
             dst_path = clone_volumes[0].path
+            start_time = datetime.now()
+            update_clone_progress(fs_handle, clone_volumes, start_time)
             bulk_copy(fs_handle, src_path, dst_path, should_cancel)
             set_quota_on_clone(fs_handle, clone_volumes)
+            end_time = datetime.now()
+            # start_time could have changed if crashed and recovered. Hence fetch from config file
+            actual_start_time = datetime.fromisoformat(clone_volumes[0].metadata_mgr.get_option("progress", "start_time"))
+            idle_time = clone_volumes[0].metadata_mgr.get_option("progress", "idle_time")
+            log.info("Clone copy time: ({0}, {1}, {2}, idle_time = {3}, completed_time = {4})".format(volname,
```

Comment:
if possible, please use `f""` format strings ... they are more intuitive to understand